src/benchmarklib/abstract_table_generator.cpp

```diff
@@ -373,11 +375,13 @@ void AbstractTableGenerator::_create_table_indexes(
     std::unordered_map<std::string, BenchmarkTableInfo>& table_info_by_name) {
   Timer timer;
   std::cout << "- Creating table indexes" << std::endl;
+
   const auto& indexes_by_table = _indexes_by_table();
   if (indexes_by_table.empty()) {
     std::cout << "-  No indexes defined by benchmark" << std::endl;
     return;
   }
+
   for (const auto& [table_name, indexes] : indexes_by_table) {
     const auto& table = table_info_by_name[table_name].table;
 
```

Comment:
Below at line 395, there is a superflous closing bracket.

```
                  << chunk_ids.size() << " (all finalized) chunks]" << std::flush;
                                                                 ^
```

Personally, I would remove the "an" in "Creating an index".
Moreover, do we need to tell how many indexes are covered here? Since it's done on the benchmark data `auto chunk_ids = std::vector<ChunkID>(table->chunk_count());`, it is always the entire table anyways.

Reply:
It's now simplified.